app/themes/dashboard/ui/app/root/root.html

```diff
@@ -1,68 +1,19 @@
+    <!-- /.search form -->
+    <!-- sidebar menu: : style can be found in sidebar.less -->
+    <ul class="sidebar-menu">
+      <li ng-repeat="link in ctrl.navService.getLinks('dashboard')" ng-class="{ active: ctrl.navService.isActive(link.state) }"><a ng-click="ctrl.navService.gotoLink(link)"><i class="fa" ng-class="link.label.navClass"></i> <span>{{ link.label.text }}</span></a></li>
     </ul>
-    <ml-user mode="inline"></ml-user>
-  </div><!-- /.navbar-collapse -->
-</nav>
-<!-- Add your site or application content here -->
-<div class=" content">
-  <div class="container">
-    <message-board msg="ctrl.messageBoardService.message()"></message-board>
-    <div ng-hide="ctrl.messageBoardService.message()">
-      <ui-view></ui-view>
-    </div>
-  </div>
-</div>
+  </section>
+</aside>
 
-<footer id="footer" class="powered">
-  <div class="container">
-    <div class="row">
-      <div class="col-sm-12">
-        <div class="social-links">
-          <a href="https://www.facebook.com/marklogic" title="Facebook" rel="external">
-            <img src="images/facebook.svg" alt="Facebook" title="Facebook"></a>
-          <a href="https://twitter.com/marklogic" title="Twitter" rel="external">
-            <img src="images/twitter.svg" alt="Twitter" title="Twitter"></a>
-          <a href="http://www.linkedin.com/company/marklogic" title="LinkedIn" rel="external">
-            <img src="images/linkedin.svg" alt="LinkedIn" title="LinkedIn"></a>
-          <a href="https://www.flickr.com/photos/marklogicuc/" title="Flickr" rel="external">
-            <img src="images/flickr.svg" alt="Flickr" title="Flickr"></a>
-          <a href="https://www.youtube.com/channel/UCT2Q5qhC2gmd89K9qpV2TrQ" title="YouTube" rel="external">
-            <img src="images/youtube.svg" alt="YouTube" title="YouTube"></a>
-          <a href="https://plus.google.com/+MarklogicCo/posts" title="Google+" rel="external">
-            <img src="images/google.svg" alt="Google+" title="Google+"></a>
-        </div>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-sm-12">
-        <ul id="menu-footer-navigation-logged-in" class="nav">
-          <li id="menu-item-140" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-140">
-            <a target="_blank" href="http://www.marklogic.com">MarkLogic.com</a>
-          </li>
-          <li id="menu-item-71" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-71">
-            <a target="_blank" href="http://developer.marklogic.com">MarkLogic Developers</a>
-          </li>
-        </ul>
-      </div>
-    </div>
-    <div class="row copyright">
-      <div class="col-sm-12 text-muted">
-        Copyright <sup>&copy;</sup> {{ ctrl.currentYear }} MarkLogic Corporation. MARKLOGIC <sup>&reg;</sup> is a registered trademark of MarkLogic Corporation.
-      </div>
-    </div>
-  </div>
-</footer>
+<div class="content-wrapper" style="min-height:800px">
+  <ui-view />
+</div>
```

Comment:
Here is the footer to add in. It is missing @grtjn's new copyright year. I'm not sure how to merge that in here. 
`<footer id="footer" class="powered">
          <div class="container">
            <div class="row">
              <div class="col-sm-12">
                <div class="social-links">
                  <a href="https://www.facebook.com/marklogic" title="Facebook" rel="external">
                    <img src="images/facebook.svg" alt="Facebook" title="Facebook"></a>
                  <a href="https://twitter.com/marklogic" title="Twitter" rel="external">
                    <img src="images/twitter.svg" alt="Twitter" title="Twitter"></a>
                  <a href="http://www.linkedin.com/company/marklogic" title="LinkedIn" rel="external">
                    <img src="images/linkedin.svg" alt="LinkedIn" title="LinkedIn"></a>
                  <a href="https://www.flickr.com/photos/marklogicuc/" title="Flickr" rel="     external">
                    <img src="images/flickr.svg" alt="Flickr" title="Flickr"></a>
                  <a href="https://www.youtube.com/channel/UCT2Q5qhC2gmd89K9qpV2TrQ" title="YouTube" rel="external">
                    <img src="images/youtube.svg" alt="YouTube" title="YouTube"></a>
                  <a href="https://plus.google.com/+MarklogicCo/posts" title="Google+" rel="external">
                    <img src="images/google.svg" alt="Google+" title="Google+"></a>
                </div>
              </div>
            </div>
            <div class="row">
              <div class="col-sm-12">
                <ul class="nav">
                  <li>
                    <a target="_blank" href="http://www.marklogic.com">MarkLogic.com</a>
                  </li>
                  <li>
                    <a target="_blank" href="http://developer.marklogic.com">MarkLogic Developers</a>
                  </li>
                </ul>
              </div>
            </div>
            <div class="row copyright">
              <div class="col-sm-12">
                Copyright <sup>&copy;</sup> 2016 MarkLogic Corporation. MARKLOGIC <sup>&reg;</sup>  is a registered trademark of MarkLogic Corporation.
              </div>
            </div>
          </div>
        </footer>`